internal/hardware/validator_test.go

```diff
@@ -964,6 +964,46 @@ var _ = Describe("hardware_validator", func() {
 		Expect(disks[8].Name).Should(Equal("nvme01fs3"))
 	})
 
+	It("validate_bootable_disk_preferred", func() {
```

Comment:
Maybe worth adding a test with multiple bootable disks and ensure they're sorted according to the other conditions.
E.g. (should return the smallest bootable)
nvme0 (bootable: true, size: 10GiB, Hctl: "N:0:1:3")
nvme1 (bootable: true, size: 2TiB, Hctl: "N:0:1:2")
nvme2 (bootable: true, size: 4TiB, Hctl: "N:0:1:1")